site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -193,7 +264,7 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 The unique Id of the identity provider.
 
 [field]#identityProviderUserId# [type]#[String]# [required]#Required#::
-The Id for the User that is provided by the identity provider. This is the value that will allow FusionAuth to link this user on future logins. It is expected to be immutable.
+The Id for the User that is provided by the upstream identity provider. This is the value that will allow FusionAuth to link this user on future logins. It must be immutable.
```

Comment:
Expected may be a better choice here- we cannot enforce immutability, this is really up to the IdP. Ideally it is immutable, but it is out of our control.

Reply:
What are the ramifications if it is not immutable? I could go with expected, because as far as I can tell, bad things might happen if it changes.

Reply:
Nothing bad will happen. (we wouldn't break) 

We just won't consider it the same link, so in a worst case example, if the IdP provided a different value each time we logged in- we'd have `n` links after `n` logins.

Reply:
Leave as is 👍 @mooreds and @robotdan